### PR TITLE
raceintospace: New recipe

### PR DESCRIPTION
**Pull request by win8linux**
*Race Into Space* (RIS) is a port to modern operating systems of *Buzz Aldrin’s Race Into Space* (BARIS), a DOS game originally released in 1993 on floppy disk, and a year later in an expanded CD version. BARIS was developed by Fritz Bronner based on his 1989 board game *Liftoff!*. Dr. Aldrin’s name and imagery have been dropped because permission to use them was limited to the old DOS game.

Screenshot of the game working on the latest `x86_64` nightly:
Screenshot_Haiku (Linked Base for Haiku and Haiku Beta)  Running  - Oracle VM VirtualBox_1

### CODE REVIEW

File: games-simulation/raceintospace/raceintospace-v2.0~git.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+PROVIDES="
+	raceintospace$secondaryArchSuffix = $portVersion
+	app:raceintospace = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	boost170$secondaryArchSuffix
+	lib:libjsoncpp$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
```
Comment (Begasus) [inline games-simulation/raceintospace/raceintospace-v2.0~git.recipe:35]: This should be libpng16 (also in devel:*)
```diff
- 	lib:libpng$secondaryArchSuffix
+ 	lib:libpng16$secondaryArchSuffix
```

File: games-simulation/raceintospace/raceintospace-v2.0~git.recipe [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	raceintospace$secondaryArchSuffix = $portVersion
+	app:raceintospace = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	boost170$secondaryArchSuffix
```
Comment (korli) [inline games-simulation/raceintospace/raceintospace-v2.0~git.recipe:33]: lib:libboost_system$secondaryArchSuffix

File: games-simulation/raceintospace/raceintospace-v2.0~git.recipe [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	lib:libprotobuf$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	lib:libtheora$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	boost170${secondaryArchSuffix}_devel
```
Comment (korli) [inline games-simulation/raceintospace/raceintospace-v2.0~git.recipe:45]: devel:libboost_system$secondaryArchSuffix >= 1.70.0

Comment (win8linux) [reply games-simulation/raceintospace/raceintospace-v2.0~git.recipe:45]: HaikuPorter fails to find this dep with this change, so this change will not be made.

**Pull request merged**